workspace-types/src/input.rs

```diff
@@ -90,6 +116,12 @@ pub struct ProofInput {
     pub proof: Vec<Vec<u8>>,
 }
 
+#[derive(Debug, Default, Eq, PartialEq, Clone, BorshSerialize, BorshDeserialize)]
+pub struct IsUsedProofCallArgs {
```

Comment:
What's the rationale for not using the same type from the engine crate directly? I saw further below that the engine type is replaced by this local type.

Reply:
We use `aurora-engine-types` crate as a dependency but `IsUsedProofCallArgs` is in the `aurora-engine`. I'm not sure if it makes sense to use `aurora-engine` for this one.